# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/src/APZCTreeManager.cpp
# Commit: fef48bed539e
# Full Hash: fef48bed539e49f3bd5859af1045da795bdab8f9
# Author: Botond Ballo <botond@mozilla.com>
# Date: 2022-09-29 21:45:56
# Description:
#   Bug 1792708 - Access aHit instead of mTouchBlockHitResult consistently in AdjustEventPointForDynamicToolbar(). r=dlrobertson
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D158287
# ==============================================================================

diff -r 990b01753f77 -r fef48bed539e gfx/layers/apz/src/APZCTreeManager.cpp
--- a/gfx/layers/apz/src/APZCTreeManager.cpp	Thu Sep 29 07:50:27 2022 +0000
+++ b/gfx/layers/apz/src/APZCTreeManager.cpp	Thu Sep 29 08:28:43 2022 +0000
@@ -2075,7 +2075,7 @@
     SideBits sideBits = SideBits::eNone;
     {
       RecursiveMutexAutoLock lock(mTreeLock);
-      sideBits = SidesStuckToRootContent(mTouchBlockHitResult.mNode.Get(lock));
+      sideBits = SidesStuckToRootContent(aHit.mNode.Get(lock));
     }
     MutexAutoLock lock(mMapLock);
     aEventPoint -= RoundedToInt(apz::ComputeFixedMarginsOffset(
